src/sentry/utils/snuba.py

```diff
@@ -39,6 +39,9 @@
     'project.id': 'project_id',
     'platform': 'platform',
     'message': 'message',
+    'title': 'title',
```

Comment:
I think these are only required for cases where the sentry and snuba names aren't the same

Reply:
Does it mean we can remove the other ones too?

Reply:
Actually, no I was wrong, this needs to contain everything that isn't a tag, because it's used like this:

```
return SENTRY_SNUBA_MAP.get(name, u'tags[{}]'.format(name))
```
so yeah, leave `title`, `location`, etc in here